docs/CNVs/CNV-methods.tex

```diff
@@ -670,125 +712,170 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
+\begin{equation}
+\bar{\Sigma}_t = \frac{2 \llangle \mathbf{B} \vSigma \rrangle_t - \llangle \vSigma \rrangle_t \llangle \mathbf{B}\rrangle_t}{\llangle \mathbf{B}\rrangle_t}.
+\end{equation}
+Plugging this result back in Eq.~\eqref{eq:psi_approx}, we find:
+\begin{equation}\label{eq:psi_approx_2}
+\Psi^\mathrm{approx}_t = \llangle \mathbf{B} \rrangle_t + \llangle \vSigma \rrangle_t - 2 \, \frac{\llangle \mathbf{B} \vSigma \rrangle_t}{\llangle \mathbf{B}\rrangle_t}.
+\end{equation}
+This solution is accurate to linear order in deviations of $\Sigma_{st}$ about $\bar{\Sigma}_t$.\\
+
+\noindent {\bf (Scheme 2) Newton iterations:} The complexity of solving Eq.~\eqref{eq:psi_stationarity} numerically is not too high given that the Hessian matrix is diagonal. Expanding $\mathcal{L}$ about $\vPsi_0$, we find:
+\begin{equation}
+\mathcal{L}(\vPsi) = \mathcal{L}(\vPsi_0) + \alpha_t \, (\Psi_t - \Psi_{0,t}) + \frac{1}{2} \, \beta_t \, (\Psi_t - \Psi_{t,0})^2 + \ldots
+\end{equation}
+where:
 \begin{align}
-\sum_s \left[ \frac{1}{\Psi_t + \Sigma_{st}} - \frac{\beta_{st}}{(\Psi_t + \Sigma_{st})^2} \right] = 0, \, \beta_{st} =\left( \vW E[\vz_s \vz_s^T] \vW^T \right)_{tt} +2 (m_t - m_{st}) \left( \vW E[\vz_s] \right)_t + (m_t - m_{st})^2
+\alpha_t &= \frac{\partial \mathcal{L}}{\partial \Psi_t} = -\frac{1}{2}\sum_s M_{st}\left[\frac{1}{\Psi_t + \Sigma_{st}} - \frac{B_{st}}{(\Psi_t + \Sigma_{st})^2}\right],\nonumber\\
+\beta_t &= \frac{\partial^2 \mathcal{L}}{\partial \Psi_t^2} = +\frac{1}{2}\sum_s M_{st}\left[\frac{1}{(\Psi_t + \Sigma_{st})^2} - \frac{2B_{st}}{(\Psi_t + \Sigma_{st})^3}\right].
 \end{align}
-Alternatively, the aforementioned approximation $\vSigma_s \rightarrow \bar{\vSigma}$ gives
-%
+The Newton's approximate solution is therefore:
 \begin{equation}
-\Psi_{tt} = \frac{1}{S} \sum_s \beta_{st} - \bar{\Sigma}_{tt}, \, {\rm (simplified)}
+\Psi_{t,1} = \Psi_{t,0} - \frac{\alpha_t(\vPsi_0)}{\beta_t(\vPsi_0)}.
 \end{equation}
-%
+One may start iterations using the result of Scheme 1 as the initial guess and continue until convergence.\\
 
-In the M step equation for $\vW$ we incorporate an automatic relevance determination (ARD) prior:
+\noindent{\bf Remark:} In practice, when sample-to-sample variables of $\Sigma$ was large (e.g. read depths varying from 50 to 1000 randomly), we noticed that the best approach was to use Brent root finding for each target. On average, 10 function calls yields to solution within a $10^{-6}$ tolerance. The Newton's method required approximately 20 evaluations of $\alpha_t$ and $\beta_t$ to convergance within the same tolerance. Also, we found that the most robust scheme was to start from $\Psi_t = 0$ rather than using Eq.~\eqref{eq:psi_approx_2}.\\
```

Comment:
sample-to-sample variables -> sample-to-sample variance

yields to solution -> yields a solution

The Newton's method -> Newton's method

Interesting that starting at $\Psi_t = 0$ was more robust, but I'm not sure if I understand why---could you expand?